app/src/main/res/layout/change_address_activity.xml

```diff
@@ -1,62 +1,104 @@
-            android:text="Continue to questions" />
+            android:paddingBottom="150dp">
+
+            <ImageView
+                android:id="@+id/image"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="150dp"
+                android:src="@drawable/ic_notifications"
+                android:visibility="gone"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent"
+                tools:ignore="ContentDescription"
+                tools:visibility="visible" />
+
+            <TextView
+                android:id="@+id/title"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/base_margin_quintuple"
+                android:textAppearance="?textAppearanceHeadline5"
+                android:visibility="gone"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/image"
+                tools:text="Are you moving? We'll join"
+                tools:visibility="visible" />
+
+            <TextView
+                android:id="@+id/subtitle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/base_margin_double"
+                android:layout_marginTop="@dimen/base_margin_double"
+                android:gravity="center"
+                android:textAppearance="?textAppearanceBody1"
+                android:visibility="gone"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/title"
+                tools:text="You’ll get a few question about your new place to calculate your new price and coverage."
+                tools:visibility="visible" />
+
+            <include
+                android:id="@+id/upcoming_address_layout"
+                layout="@layout/upcoming_address_change_layout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginHorizontal="@dimen/base_margin_double"
+                android:visibility="gone"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@id/subtitle" />
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+    </androidx.core.widget.NestedScrollView>
+
+    <include
+        android:id="@+id/spinner"
+        layout="@layout/loading_spinner" />
+
+    <com.google.android.material.button.MaterialButton
```

Comment:
```suggestion
    <Button
```

`<MaterialButton />` automatically gets used when using a `MaterialTheme` 😉

Reply:
Changed because I wanted to be able to access `button.icon` in the activity

Reply:
Oh! Hm 🤔 Perhaps that's an intentionally hidden implementation detail? Confusing to say the least.

Reply:
No, it's very clearly a part of the exposed API. Fair enough. I'm satisfied with keeping the code as is!